### PR TITLE
Prepare 1.1.3 release

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

See title.
Also includes a #5161 hot-fix ~and removes a duplicate import statement~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,14 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [1.1.3] - 2021-01-05
+## [1.1.4] - YYYY-MM-DD
```
Comment (Borda) [inline CHANGELOG.md:8]: let's remove this section for now and add it after release...

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,11 +21,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
-- Skip restore from `resume_from_checkpoint` in while `testing` ([#5161](https://github.com/PyTorchLightning/pytorch-lightning/pull/5161))
 
-- Allowed `log_momentum` for adaptive optimizers in `LearningRateMonitor` ([#5333](https://github.com/PyTorchLightning/pytorch-lightning/pull/5333))
+## [1.1.3] - 2021-01-05
 
-- Disabled checkpointing, earlystopping and logger with `fast_dev_run` ([#5277](https://github.com/PyTorchLightning/pytorch-lightning/pull/5277))
+### Added
+
+- Added a check for optimizer attached to lr_scheduler ([#5338](https://github.com/PyTorchLightning/pytorch-lightning/pull/5338))
```
Comment (Borda) [inline CHANGELOG.md:29]: have you check commit history and add missing, I several changes/fixes which shall be mentioned
https://github.com/PyTorchLightning/pytorch-lightning/commits/master
from this chlog we also generate user mentioning for release notes...

**Review by Borda — CHANGES_REQUESTED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Comment by Borda** [comment]
seems to be the same problem - https://stackoverflow.com/questions/50480689/pytorch-torchvision-brokenpipeerror-errno-32-broken-pipe

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -16,26 +16,26 @@ If we didn't discuss your PR in Github issues there's a high chance it will not
...
+- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
+- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
 - [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
-- [ ] Did you make sure to update the documentation with your changes [if needed]?
-- [ ] Did you write any new necessary tests [no need for typos, docs]? 
+- [ ] Did you make sure to update the documentation with your changes? (if necessary)
+- [ ] Did you write any new necessary tests? (not for typos and docs)
 - [ ] Did you verify new and existing tests pass locally with your changes?
-- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?
+- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal changes)
```
Comment (Borda) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorins)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,27 +9,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
-- Allowed `log_momentum` for adaptive optimizers in `LearningRateMonitor` ([#5333](https://github.com/PyTorchLightning/pytorch-lightning/pull/5333))
-
-- Disabled checkpointing, earlystopping and logger with `fast_dev_run` ([#5277](https://github.com/PyTorchLightning/pytorch-lightning/pull/5277))
+- Fixed `trainer.test` returning non-test metrics ([#5214](https://github.com/PyTorchLightning/pytorch-lightning/pull/5214))
+- Fixed metric state reset ([#5273](https://github.com/PyTorchLightning/pytorch-lightning/pull/5273))
+- Fixed `--num-nodes` on `DDPSequentialPlugin` ([#5327](https://github.com/PyTorchLightning/pytorch-lightning/pull/5327))
+- Fixed invalid value for `weights_summary` ([#5296](https://github.com/PyTorchLightning/pytorch-lightning/pull/5296))
+- Fixed `Trainer.test` not using the latest `best_model_path` ([#5161](https://github.com/PyTorchLightning/pytorch-lightning/pull/5161))
+- Fixed existence check for hparams not using underlying filesystem ([#5250](https://github.com/PyTorchLightning/pytorch-lightning/pull/5250))
+- Fixed `LightningOptimizer` AMP bug ([#5191](https://github.com/PyTorchLightning/pytorch-lightning/pull/5191))
```
Comment (Borda) [inline CHANGELOG.md:31]:
```suggestion
```

**Pull request merged**